mist-lib/src/services/stp-utils.tests.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-magic-numbers, no-undefined */
+'use strict'
+
+const path = require('path')
+
+const tableFilePath = path.resolve(__dirname, 'test-data/mist3-sig-trans-2017.test.csv')
+const stpUtils = require('./stp-utils.js')
+
+describe('Signal Transduction Service', function() {
```

Comment:
This is not testing the signal transduction service, but rather stp-utils. Also, please use arrow functions (e.g. `describe('stp-utils', () => { ... })`.